catalog/spatial/wfs/2.0.0/spatial-wfs-v2_0_0-connectedsource/pom.xml

```diff
@@ -112,7 +118,7 @@
         <dependency>
             <groupId>org.jvnet.jaxb2_commons</groupId>
             <artifactId>jaxb2-basics-runtime</artifactId>
-            <version>0.11.0</version>
+            <version>0.6.0</version>
```

Comment:
why the downgrade?

Reply:
DDF is currently using version 1.0.1 of ddf-jaxb. Its spatial-wfs-v2_0_0-schema-bindings artifact depends on version 1.1.0 of filter-v_2_0_0-schema, which depends on version 0.6.0 of jaxb2-basics-runtime: https://mvnrepository.com/artifact/org.jvnet.ogc/filter-v_2_0_0-schema/1.1.0.

Using version 0.11.0 results in some subtle errors like
```
17:21:57,526 | DEBUG | pool-39-thread-1 | .v2_0_0.catalog.source.WfsFilterDelegate 1729 | al-wfs-v2_0_0-source | Adding key [spatialOp Name: Disjoint]
SLF4J: Failed toString() invocation on an object of type [java.util.concurrent.ConcurrentHashMap]
Reported exception:
java.lang.NoSuchFieldError: INSTANCE
	at net.opengis.filter.v_2_0_0.SpatialOperatorType.toString(SpatialOperatorType.java:125)
	at java.lang.String.valueOf(String.java:2994)
	at java.lang.StringBuilder.append(StringBuilder.java:131)
	at java.util.concurrent.ConcurrentHashMap.toString(ConcurrentHashMap.java:1321)
	at org.slf4j.helpers.MessageFormatter.safeObjectAppend(MessageFormatter.java:299)
	at org.slf4j.helpers.MessageFormatter.deeplyAppendParameter(MessageFormatter.java:271)
	at org.slf4j.helpers.MessageFormatter.arrayFormat(MessageFormatter.java:233)
	at org.slf4j.helpers.MessageFormatter.arrayFormat(MessageFormatter.java:173)
	at org.slf4j.helpers.MessageFormatter.format(MessageFormatter.java:124)
	at org.ops4j.pax.logging.slf4j.Slf4jLogger.debug(Slf4jLogger.java:292)
	at org.codice.ddf.spatial.ogc.wfs.v2_0_0.catalog.source.WfsFilterDelegate.setSpatialOps(WfsFilterDelegate.java:1731)
```

The version mismatch might even cause more issues unrelated to logging that I haven't encountered.

Reply:
spatial-wfs-v2_0_0-source is already using version 0.6.0